### PR TITLE
Updated data backup README

### PR DESCRIPTION
**Pull request by anitamegarry**

### CODE REVIEW

File: data-backup/README.md [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
+
+### Dockerisation
+
+A docker image can be created with `dockerfile`. On activation, this will install all the requirements and run the `extract_from_database.py` file in a controlled environment.
+
+To create and use the Docker image, follow these steps:
+
+1. Build the Docker image:
+
+docker build -t data-backup .
```
Comment (jiuliangut) [inline data-backup/README.md:28]: do we need to mention to build it with --platform linux/amd64 ?

**Review by jiuliangut — COMMENTED** [review]
nice and comprehensive!

**Review by jiuliangut — APPROVED** [review]
looks good!

**Review by Fahi28 — APPROVED** [review]
good stuff

**Pull request merged**